internal/errors/error.go

```diff
@@ -85,6 +85,10 @@ func (e *wrapError) As(target interface{}) bool {
 	return xerrors.As(err, target)
 }
 
+func (e *wrapError) Unwrap() error {
+	return e.nextErr
```

Comment:
Maybe, I think need chaining `nextErr` to get the root error instance like the following .

```go
err := e.nextErr
for {
  if wrapErr, ok := err.(*wrapError); ok {
    err = wrapErr.nextErr
    continue
  }
  break
}
return err
```

Reply:
I think if the `nextErr` is `*wrapError`, `errors.Is` calls `(*wrapError).Unwrap` recursively so current implementation has no problem
if for users not using `errors.Is`, `Unwrap` sholud return just its parent, not recursively, HDYT

Reply:
Oh, You're right ! Thanks !